### ISSUE TITLE
property definitions

### ISSUE THREAD

**Issue by wdduncan** [mdebellis/DrMO_Docs#5]
I've notice the object, data, and annotation properties don't have definitions.

**Comment by mdebellis** [comment]
Nivedita, when we do the next version of the ontology after you finish your
paper, we should address this. BTW, I think we can also vectorize these
definitions and use them to give answers as well as the corpus so there is
even more reason to do this besides it is a best practice.

**Comment by wdduncan** [comment]
Yes @mdebellis ! I was thinking about vectorization when I made the issue ;)
